styles.css

```diff
@@ -902,14 +902,15 @@ form {
     margin-top: 12px;
     margin-bottom: 12px;
   }
+
   .know-text {
     list-style-type: none;
   }
 
   #languages {
     margin-top: 80px;
   }
-  
+
   .know-line {
```

Comment:
- [ ] Kindly adjust the margin between the inputs in your `form section` as the margins currently are bigger compared to the original design.

![image](https://user-images.githubusercontent.com/98453979/220944743-269f6b46-f79a-4c3c-80a0-459da67f8d4f.png)

### Checkout the original design
![image](https://user-images.githubusercontent.com/98453979/220944950-37694efd-384e-4cb2-ace2-d914d825c7c2.png)